src/plugins/irc-events/link.js

```diff
@@ -93,13 +93,8 @@ function fetch(url, cb) {
 		return;
 	}
 	var length = 0;
-	var limit = 1024 * 10;
+	var limit = Helper.config.prefetchMaxImageSize * 1024;
```

Comment:
Because we only need to fetch some meta tags from the page, which are at the top, so 10kb works fine for that purpose.